frostsnapp/lib/main.dart

```diff
@@ -196,7 +197,23 @@ class _MyHomePageState extends State<MyHomePage> {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-        appBar: AppBar(title: Text(widget.title)),
+        appBar: AppBar(title: Text(widget.title), actions: [
+          PopupMenuButton<String>(
+            onSelected: (String result) {
+              if (result == 'feedback') {
+                Navigator.push(context, MaterialPageRoute(builder: (context) {
+                  return FeedbackPage();
```

Comment:
I wonder if an alert dialog would make more sense here. Once you submit you can dismiss it. With the page you can do this too but it currently does it and it's less idiomatic.